lib/ui/painting/fragment_program.cc

```diff
@@ -72,7 +72,7 @@ fml::RefPtr<FragmentShader> FragmentProgram::shader(
   uniforms.Release();
   std::vector<sk_sp<SkShader>> sk_samplers(sampler_shaders.size());
   for (size_t i = 0; i < sampler_shaders.size(); i++) {
-    SkSamplingOptions sampling;
+    DlImageSampling sampling = DlImageSampling::kDefault;
```

Comment:
The comment below this line about using "the default value for SkSamplingOptions" is confusing. Minimally the reference to SkSamplingOptions needs to be removed, but overall I think the following would make a better comment:

// ImageShaders can hold a preferred value for sampling options and developers
// are encouraged to use that value or the value will be supplied by "the environment
// where it is used". The environment here does not contain a value to be used if
// the developer did not specify a preference when they constructed the ImageShader,
// so we will use kNearest which is the default filterQuality in a Paint object.